src/CartItemList.tsx

```diff
@@ -50,66 +52,82 @@ export const CartItemList: React.FC<CartItemListParams> = (props) => {
   };
 
   return (
-      <div className={`cartitemlist ${isLoading ? '--loading' : ''}`}>
-        <h2 className="cartitemlist__title">
-          {t('your-shopping-cart')}
-        </h2>
-        {items && items.length > 0 ? (
-          <div>
-            <div className="cartitemlist__wrap">
-              {items.map((item: any, index:number) => (
-                <div key={item.id} className={`cartitemlist__product ${removingItem === index ? '--removing' : ''}`} >
-                  <div className="cartitemlist__image">
-                    {item.image && item.image.href && (
-                      <ImageContainer
-                        imgClassName="productmainimage"
-                        imgUrl={item.image.href}
-                        alt={item.image.name}
-                        imageStyle={{ width: imgSize, height: imgSize, objectFit: 'fill', backgroundColor: '' }}
-                      />
-                    )}
+    <div className={`cartitemlist ${isLoading ? '--loading' : ''}`}>
+      {isLoggedIn && (
+        <button className="cartitemlist__mycart" onClick={() => onHandlePage('cartsList')}>
+          {t('my-carts')}
+        </button>
+      )}
+      <h2 className="cartitemlist__title">
+        {isLoggedIn && items.length ? (
+          <span>
+            {selectedCartName}
+            {/*<span className="cartitemlist__settingsicon">*/}
```

Comment:
leftover commented code

Reply:
@shaunmaharaj It's Cart Edit button, I commented it because API isn't available for now